server/services/AlertService.js

```diff
@@ -75,7 +75,8 @@ export default class AlertService {
     params.alertState = alertState;
     params.searchString = search;
     if (search.trim()) params.searchString = `*${search.trim().split(' ').join('* *')}*`;
-    if (monitorIds.length > 0) params.monitorId = monitorIds[0];
+    if (monitorIds.length > 0 && !Array.isArray(monitorIds))
+      params.monitorId = !Array.isArray(monitorIds) ? monitorIds : monitorIds[0];
```

Comment:
Redundant check for `!Array.isArray(monitorIds)` since it's already done as part of the `if` check.

Reply:
thanks for catching that